src/components/MoneyRequestConfirmationList.tsx

```diff
@@ -545,7 +545,7 @@ function MoneyRequestConfirmationList({
         let updatedTagsString = TransactionUtils.getTag(transaction);
         policyTagLists.forEach((tagList, index) => {
             const enabledTags = Object.values(tagList.tags).filter((tag) => tag.enabled);
-            const isTagListRequired = tagList.required === undefined ? false : tagList.required && canUseViolations;
+            const isTagListRequired = tagList.required ?? false;
```

Comment:
We need to remove `canUseViolations` from useEffect deps.

Reply:
Also, we need to remove it from [here](https://github.com/Expensify/App/blob/8787a6464e6d2f75bff5433be63b3c663431c4fb/src/components/MoneyRequestConfirmationList.tsx#L880) to ensure it will be out "show more" section when been auto selected, also add it in `rightLabel` like what we do above in `category`